Sources/PostgresNIO/New/PostgresCell.swift

```diff
@@ -1,14 +1,26 @@
 import NIOCore
 
+/// A representation of a cell value within a ``PostgresRow`` and ``PostgresRandomAccessRow``.
 public struct PostgresCell: Equatable {
+    /// The cell's value as raw bytes.
     public var bytes: ByteBuffer?
+    /// The cell's data type. This is important metadata when decoding the cell.
     public var dataType: PostgresDataType
+    /// The format in which the cell's bytes are encoded
```

Comment:
To keep it consistent
```suggestion
    /// The format in which the cell's bytes are encoded.
```